ide/markdown/src/org/netbeans/modules/markdown/ui/preview/views/ViewUtils.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.netbeans.modules.markdown.ui.preview.views;
+
+import javax.swing.text.AbstractDocument;
+import javax.swing.text.AttributeSet;
+import javax.swing.text.Element;
+import javax.swing.text.StyleConstants;
+import javax.swing.text.html.HTML;
+
+/**
+ * An utility class to deal with HTML elements.
+ *
+ */
+public final class ViewUtils {
+
+    public static boolean isElementOfTag(Element elem, HTML.Tag tag) {
```

Comment:
this is just a nitpick but I try to follow the rule to not create utility classes unless the code is used from at least two different call sites. Otherwise the class count would explode quickly.

but I do get that you probably didn't want to have this in the factory and there are probably more components to add in future.

Reply:
No problem, you are right, we can put this method isElementOfTag directly LocalViewFactory and delete ViewUtils, if necessary we recreate in the future.
I will fix that.
thanks

Reply:
cool! don't forget to squash.